frontend/appflowy_flutter/lib/workspace/presentation/settings/pages/settings_manage_data_view.dart

```diff
@@ -375,6 +378,8 @@ class _CurrentPathState extends State<_CurrentPath> {
 
   @override
   Widget build(BuildContext context) {
+    final isLM = Brightness.light == Theme.of(context).brightness;
```

Comment:
@LucasXu0 Create an extension to retrieve the current brightness in the next version.